test/unit/video.test.js

```diff
@@ -252,24 +245,7 @@ QUnit.test('should expose DOM functions', function(assert) {
   methods.forEach(name => {
     assert.strictEqual(typeof videojs[name], 'function', `function videojs.${name}`);
     assert.strictEqual(typeof Dom[name], 'function', `Dom.${name} function exists`);
-
-    const oldMethod = Dom[name];
```

Comment:
I don't think we needed to check if dom methods were called via deprecated videojs properties. I think its enough just to check that they exist.